core/src/main/java/org/infinispan/configuration/parsing/ParserRegistry.java

```diff
@@ -229,7 +243,7 @@ private NamespaceParserPair findNamespaceParser(String namespace, String name) {
          parser = parserMappings.get(new QName(baseUri, name));
          // See if we can get a default parser instead
          if (parser == null || !isSupportedNamespaceVersion(parser.namespace, namespace.substring(lastColon + 1)))
-            throw CONFIG.unsupportedConfiguration(name, namespace, Version.getVersion());
+            return null;
```

Comment:
I replaced that throw statement on another step forward. That's because the cache name does not follow any namespace definition in YAML and JSON. In both cases, the cache name is the root key of all the cache definition. So I throw the same statement when I verify if the root is not valid and the next child is not a cache type definition. That verification is in ParseRegistry.java line 216

Reply:
Refactored using parseCacheName when it's not in the namespace definition